src/compat/object/fromEntries.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Transforms an array of key-value pairs into an object.
+ *
+ * @param {Array} data - An array of key-value pairs.
+ * @returns {Record<T, U>} - An object with keys and values from the input array.
+ *
+ * @example
+ * const entries = [['a', 1], ['b', 2]];
+ *
+ * const object = fromEntries(entries);
+ * console.log(object);
+ * // Output:
+ * // {
+ * //   a: 1,
+ * //   b: 2
+ * // }
+ */
+
+export function fromEntries<T extends PropertyKey, U>(
```

Comment:
Oh! Why not we just name this as `fromPairs` as in lodash, since it is for compatibility purposes?

Reply:
I think the "fromPairs" currently can be considered as part of the JavaScript syntax.but I can change the name if it's not standard.